src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml

```diff
@@ -11,14 +11,14 @@
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
 				<MaterialColors xmlns="using:Uno.Material"
-								ColorPaletteOverrideSource="ms-appx:///ColorPaletteOverride.xaml" />
+								OverrideSource="ms-appx:///ColorPaletteOverride.xaml" />
```

Comment:
Update docs for this.

Reply:
https://github.com/unoplatform/uno/pull/7464